app/controllers/concerns/formatting_concern.rb

```diff
@@ -6,21 +6,22 @@ module FormattingConcern
   def records_to_xml_string(records)
     # TODO: Re-enable. Disabled as we no longer have VoyagerHelpers.
     # if records.kind_of? Array
-    #   xml_str = ''
-    #   StringIO.open(xml_str) do |io|
-    #     writer = MARC::XMLWriter.new(io)
-    #     records.each do |r|
-    #       writer.write(r) unless r.nil?
-    #     end
-    #     writer.close()
-    #   end
-    #   VoyagerHelpers::Liberator.valid_xml(xml_str)
-    # elsif records.kind_of? String
-    #   # example response from /almaws/v1/bibs/{mms_id}/holdings
-    #   valid_xml(records)
-    # else
-    #   valid_xml(records.to_xml.to_s)
-    # end
+    #           xml_str = ''
+    #           StringIO.open(xml_str) do |io|
+    #             writer = MARC::XMLWriter.new(io)
+    #             records.each do |r|
+    #               writer.write(r) unless r.nil?
+    #             end
+    #             writer.close()
+    #           end
+    #           valid_xml(xml_str)
+    #         elsif records.kind_of? String
+    #           # example response from /almaws/v1/bibs/{mms_id}/holdings
+    #           valid_xml(records)
+    #         else
+    #           valid_xml(records.to_xml.to_s)
+    records.to_xml.to_s
+    #         end
```

Comment:
I didn't test the valid_xml and didn't include it. We will need to find an xml with the exceptions listed in the valid_xml method or different ones.